.gitignore

```diff
@@ -1,2 +1,7 @@
 target/
-.vscode
\ No newline at end of file
+.vscode
+node_modules
+typechain-types
+artifacts
+cache
+.DS_Store
```

Comment:
Please remove these files from git too

Reply:
and also it seems like you need to ignore these folders like for example `**/node_modules`